src/index.js

```diff
@@ -1,34 +1,23 @@
 import './styles/Sass/main.sass';
+import { addScore } from './modules/api.js';
+import { user, score, renderList } from './modules/displayList.js';
 
-const body = document.querySelector('body');
+// const body = document.querySelector('body');
+const submitBtn = document.getElementById('submit');
+// const gameUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/9W40RRKEzdVGseFky5fH/scores/';
 
-const layout = () => {
-  body.innerHTML = `
-    <header><h1>Leaderboard</h1></header>
-    <main>
-      <section>
-        <div class="Side">
-          <div class="fstSect">
-            <h3>Recent Scores</h3>
-            <button class="btn">Refresh</button>
-          </div>
-          <ul class="recentScores">
-            <li>Name:96</li>
-            <li>Name: 87</li>
-            <li>Name:100</li>
-          </ul>
-        </div>
-        <div class="Side">
-          <h3>Add your score</h3>
-          <input type="text" /><input type="text" />
-          <button class="btn">Submit</button>
-        </div>
-      </section>
-    </main>
-    <footer>
-      <h3 class="copyRights">Copy rights reserved to <a href="https://github.com/GeekyHacks">geekyhacks</a></h3>
-    </footer>
-    `;
-};
+window.addEventListener('load', () => {
+  renderList();
+});
+// this will add userScore to the APi
+submitBtn.addEventListener('click', async (event) => {
+  event.preventDefault();
+  await addScore(user.value, score.value);
+});
 
-layout();
+// will use it next project
+// const refresh = document.getElementById('refresh');
+
+// refresh.addEventListener('click', async () => {
+//   await deleteAPI(gameID);
+// });
```

Comment:
- [ ] It is unprofessional to leave commented code in your codebase so I kindly suggest you remove these highlighted lines of code. 👍 ✨